aws/resource_aws_ebs_default_kms_key.go

```diff
@@ -0,0 +1,80 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsEbsDefaultKmsKey() *schema.Resource {
+	return &schema.Resource{
```

Comment:
We should add import support from the get-go, e.g. via the key ARN as the ID 🚀 

```go
Importer: &schema.ResourceImporter{
	State: schema.ImportStatePassthrough,
},
```

In the testing:

```go
{
	ResourceName:      resourceName,
	ImportState:       true,
	ImportStateVerify: true,
},
```

And the documentation:

``````md
## Import

EBS Default KMS Key can be imported with the KMS Key ARN, e.g.

```console
$ terraform import aws_ebs_default_kms_key.example arn:aws:kms:us-east-1:123456789012:key/abcd-1234
```
``````

Reply:
Since `resourceAwsEbsDefaultKmsKeyRead()` doesn't actually use the resource's ID for the AWS API call (there's a single default EBS CMK per region per account) do we want to check that the KMS key ARN passed to `terraform import` is really the current default EBS CMK?

Reply:
If the Terraform code corresponding to the imported resource has the same `key_arn` value as the ARN passed to `terraform import` then I think that having `ForceNew` on the `key_arn` attribute will cause the resource to be recreated.